tests/query/injections/svelte/test-svelte-injections.svelte

```diff
@@ -5,6 +5,7 @@
 <script lang="ts">
   const foo: number = 1
 //            ^ typescript
+//            ^ !javascript
```

Comment:
can you test whether the test suite for injections supports negative assertions?

Reply:
Yeah looks like it doesn't support negative assertion just yet, will add this to injection spec then